parceler/src/main/java/org/parceler/internal/matcher/RemoterMatcher.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2011-2015 John Ericksen
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.parceler.internal.matcher;
+
+
+import org.androidtransfuse.adapter.ASTStringType;
+import org.androidtransfuse.adapter.ASTType;
+import org.androidtransfuse.util.matcher.Matcher;
+
+/**
+ * Matches a @Remoter inteface
+ *
+ * @author js
+ */
+public class RemoterMatcher implements Matcher<ASTType> {
+
+    private static final ASTType REMOTER = new ASTStringType("remoter.annotations.Remoter");
```

Comment:
Why aren't your projects under com.josesamuel matching your groupid on maven central?

Reply:
Here is the github https://github.com/josesamuel/remoter. 

I am not sure how to match that with the group id....New to all these.....I thought it is similar to how org.parceler is at  https://github.com/johncarl81/parceler

Reply:
I would change the package structure of your project to match the maven group id.  Like this: `com.josesamuel.remoter.annotations.Remoter`.  I'm not sure where I read about this before, and it's certainly not a rule, but maybe just a naming convention.  Found an article here about it: http://blog.sonatype.com/2011/01/maven-tip-project-directories-and-artifact-ids/

Reply:
Thanks for the info, i will keep that in mind for future projects. It is too late to change it for remoter, as it is already in use.

Reply:
I actually followed the example of Jake Wharton's butter knife https://github.com/JakeWharton/butterknife

Reply:
Fair enough.